diff --git a/node_modules/react-native-blurhash/android/src/main/java/com/mrousavy/blurhash/BlurhashViewManager.kt b/node_modules/react-native-blurhash/android/src/main/java/com/mrousavy/blurhash/BlurhashViewManager.kt
index 0000000..1111111 100644
--- a/node_modules/react-native-blurhash/android/src/main/java/com/mrousavy/blurhash/BlurhashViewManager.kt
+++ b/node_modules/react-native-blurhash/android/src/main/java/com/mrousavy/blurhash/BlurhashViewManager.kt
@@ -108,12 +108,12 @@ class BlurhashViewManager :
     return image
   }

-  override fun getExportedCustomDirectEventTypeConstants(): MutableMap<String, Any>? {
-    return MapBuilder.builder<String, Any>()
-        .put(LoadErrorEvent.EVENT_NAME, MapBuilder.of("registrationName", "onLoadError"))
-        .put(LoadStartEvent.EVENT_NAME, MapBuilder.of("registrationName", "onLoadStart"))
-        .put(LoadEndEvent.EVENT_NAME, MapBuilder.of("registrationName", "onLoadEnd"))
-        .build()
+  override fun getExportedCustomDirectEventTypeConstants(): MutableMap<String, Any?>? {
+    return mutableMapOf<String, Any?>(
+        LoadErrorEvent.EVENT_NAME to mapOf("registrationName" to "onLoadError"),
+        LoadStartEvent.EVENT_NAME to mapOf("registrationName" to "onLoadStart"),
+        LoadEndEvent.EVENT_NAME to mapOf("registrationName" to "onLoadEnd"),
+    )
   }

   override fun getName() = REACT_CLASS
